guide/inheritance-wallet/landing-page.md

```diff
@@ -50,142 +50,38 @@ All of this is done without compromising your financial privacy. You don't want
 
 ### [Overview]({{ '/guide/inheritance-wallet/overview/' | relative_url }})
 
-<div class="center" markdown="1">
-
-{% include image.html
-   image = "assets/images/guide/inheritance-wallet/icon-introduction.png"
-   retina = "assets/images/guide/inheritance-wallet/icon-introduction@2x.png"
-   alt-text = ""
-   width = 80
-   height = 80
-   layout = "float-left"
-   link-url = "/guide/inheritance-wallet/introduction/"
-%}
-
 Before we dive into the details, we will introduce you to the Jones family. We are also going to look at how they plan to set up their family savings in general. 
 
-</div>
-
 ---
 
 ### [Wallet creation]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }})
 
-<div class="center" markdown="1">
+Our application lets users create flexible multi-key wallets with timelocked recovery paths to help them recover funds while they are still alive. On top of that, users can create and manage dedicated inheritance keys that are held by their heirs.
```

Comment:
```suggestion
Our application lets users create flexible multi-key wallets with timelocked recovery paths to help them recover funds. On top of that, users can create and manage dedicated inheritance keys that are held by their heirs.
```
It does not feel necessary to add in while they are alive here since if someone is dead they wont be using a wallet :-)